### ISSUE TITLE
Google_Utils_UriTemplate - URITemplate.php not autoloading in Resource.php 

### ISSUE THREAD

**Issue by viction** [google/google-api-php-client#920]
>PHP Fatal error:  Class 'Google_Utils_UriTemplate' not found in /var/www/jp/current/vendor/google/apiclient/src/Google/Service/Resource.php on line 286

Is this related to removing Google_Utils from this release?

My hack is to explicitly require_once URITemplate.php in Resource.php... is there is a cleaner fix? 

using php 5.5.9 
installed library in my project using composer.

trace stack:

>[05-Apr-2016 11:21:49 America/Chicago] PHP Fatal error:  Class 'Google_Utils_UriTemplate' not found in /var/www/jp/current/vendor/google/apiclient/src/Google/Service/Resource.php on line 286
>[05-Apr-2016 11:21:49 America/Chicago] PHP Stack trace:
>...
>[05-Apr-2016 11:21:49 America/Chicago] PHP   8. RCP_Social_Google->getUser() /var/www/jp/current/application/modules/member/controllers/LoginController.php:407
>[05-Apr-2016 11:21:49 America/Chicago] PHP   9. Google_Service_Plus_People_Resource->get() /var/www/jp/current/library/RCP/Social/Google.php:139
>[05-Apr-2016 11:21:49 America/Chicago] PHP  10. Google_Service_Resource->call() /var/www/jp/current/vendor/google/apiclient/src/Google/Service/Plus.php:419
>[05-Apr-2016 11:21:49 America/Chicago] PHP  11. Google_Service_Resource->createRequestUri() /var/www/jp/current/vendor/google/apiclient/src/Google/Service/Resource.php:192

**Comment by bshaffer** [comment]
What was your solution?

**Comment by viction** [comment]
The most recent composer update seemed to fix my issue... Google_Utils_UriTemplate auto loads as expected now.  Sorry I don't have more insight.

**Comment by viction** [comment]
@bshaffer, I wanted to follow-up with my definitive solution.  

Everything was working in my test/stage/production environments, but my dev continued to have issues with the auto_include without the explicit require hack I outlined originally.

I discovered that somewhere along the way, the UriTemplate.php file was named/renamed URITemplate.php -- therefore, the autocomplete had issues finding the file because of case discrepancy even though the class name was correct in that file.

Not sure why composer didn't update/fine/fix the file name? - regardless, now works as expected in all my environments.
